docs-site/components/RtcManager.md

```
@@ -15,6 +18,32 @@ The RTC Manager component interfaces with the Real Time Clock (RTC) to provide t
 
+#### `ALARM_SET` Command Usage
+1. The component is instantiated and initialized during system startup
+2. A ground station sends a `ALARM_SET` command with the desired time
+3. On each command, the component:
+    - Validates that the alarm is at a future date
+    - Emits 'AlarmNotSet' if time is not valid or if an alarm is already present
+    - Sets the time to the RTC alarm if validation passes
+    - Emits a `AlarmSet` event with the previous time if the alarm is set successfully
+    - Emits a `AlarmNotSet` event if the alarm is not set successfully
+
+#### `ALARM_CANCEL` Command Usage
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)